Overview.src.html

```diff
@@ -2490,6 +2496,25 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
  <li><p>Let <var>contentLengthValue</var> be null.
 
+ <li>
+   <p>If <var>httpRequest</var>'s <span title=concept-request-attached-credential>attached
+   credential</span> is <em>not</em> null, run these substeps:
+
+   <ol>
+     <li>If <var>httpRequest</var>'s <span title=concept-request-body>body</span> is <em>not</em>
+     null, return a <span title=concept-network-error>network error</span>.
```

Comment:
I don't think it can, but it's not clear how to write an `ASSERT(x)` in Fetch-code. :)